fabtests/pytest/efa/test_efa_protocol_selection.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+
+@pytest.mark.serial
+@pytest.mark.functional
+@pytest.mark.parametrize("fabtest_name,cntrl_env_var", [("fi_rdm_tagged_bw", "FI_EFA_INTER_MIN_READ_MESSAGE_SIZE"), ("fi_rma_bw", "FI_EFA_INTER_MIN_READ_WRITE_SIZE")])
+def test_transfer_with_read_protocol(cmdline_args, fabtest_name, cntrl_env_var):
+    """
+    Verify that the read protocol is used for a 1024 byte message when the env variable
+    switches are set to force the read protocol at 1000 bytes.
+    """
+    import copy
+    from common import has_cuda, has_hmem_support
+    from efa.efa_common import efa_run_client_server_test, efa_retrieve_hw_counter_value
+
+    if cmdline_args.server_id == cmdline_args.client_id:
+        pytest.skip("No read for intra-node communication")
+
+    if (not has_hmem_support(cmdline_args, cmdline_args.client_id) or
```

Comment:
If we are only verifying the protocol selection logic for hmem, the test name needs some change to reflect it . The current name makes me think it's testing general selection logic including host.

Reply:
done